man/man1/ztest.1

```diff
@@ -147,6 +147,12 @@ ztest -f / -V -T 120
 
 .SH "ENVIRONMENT VARIABLES"
 .TP
+.B "ZFS_HOSTID=id"
+Use \fBid\fR instead of the SPL hostid to identify this host.  Intended for use
+with ztest, but this environment variable will affect any utility which uses
+libzpool, including \fBzpool(8)\fR.  Since the kernel is unaware of this setting
```

Comment:
The _why_ in this case is because on `zpool_find_import_impl()` uses Illumos's thread pool library to perform device scanning in parallel.  Under Linux their isn't an equivalent thread pool library so we have to port this from Illumos.  Rather than doing that it was expedient to borrow the existing taskq implementation from libzpool and [use that instead](https://github.com/zfsonlinux/zfs/blob/master/cmd/zpool/zpool_main.c#L2554).

Determining if it was worth porting Illumos's thread pool library was left as future work.  Looking at `thread_pool.c` from Illumos I see that it's already using pthreads so I imagine it would be straight forward to port.  It's just a matter of doing the work.